release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -5962,3 +5962,102 @@ openshift3/ose-template-service-broker:v3.11.705-1.g7a17a5d
 ==== Upgrading
 
 To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest release, see xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrade methods and strategies] for instructions.
+
+[[ocp-3-11-715]]
+=== RHBA-2022:5000 - {product-title} 3.11.715 bug fix and security update
```

Comment:
3.11 releases are kind of backwards. This should actually be the RPM live ID

```suggestion
=== RHSA-2022:4999 - {product-title} 3.11.715 bug fix and security update
```

Reply:
Addressed